src/components/Page/LandingPageWidgets/MembersWidget.vue

```diff
@@ -100,11 +101,11 @@ export default {
 	},
 
 	mounted() {
-		window.addEventListener('resize', this.updateShowMembersCount)
+		window.addEventListener('resize', this.updateShowMembersCountDebounced)
```

Comment:
I'm not sure if this will work. I'd expect the function to be called without it being bound to `this`. So this inside the function would be undefined.

Reply:
I did some testing around the resize listener because it indeed was broken (already before this change). Using `this` inside the debounced function seems to work well though.

I'll open a follow-up PR soon that fixes the broken resize logic soon.

Reply:
You're right. I stepped through it and `this.updateShowMembersCound` is bound to `this`. I suspect that vue already binds the methods to `this`.